# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/svg/types/scripted/SVGLength-ic.html.ini
# Commit: 8a94c155e35c
# Full Hash: 8a94c155e35c69a524e0ecb17674bb5afa9c5ad3
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-09-12 16:30:15
# Regressor Bug: 1287054
# File Overlap Count: 1
# Description:
#   Bug 1287054 part 1 - support rem, ch, ic and cap units for non-CSS lengths r=emilio,jgilbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D180618
# ==============================================================================

diff -r cd564dee524a -r 8a94c155e35c testing/web-platform/meta/svg/types/scripted/SVGLength-ic.html.ini
--- a/testing/web-platform/meta/svg/types/scripted/SVGLength-ic.html.ini	Mon Sep 11 23:42:37 2023 +0000
+++ b/testing/web-platform/meta/svg/types/scripted/SVGLength-ic.html.ini	Mon Sep 11 23:50:11 2023 +0000
@@ -1,8 +1,3 @@
 [SVGLength-ic.html]
   expected:
     if (os == "android") and fission: [OK, TIMEOUT]
-  [ic unit in SVGLength]
-    expected: FAIL
-
-  [Convert back to ic from new user unit value]
-    expected: FAIL